interface/src/common/statusBar/statusBar.js

```diff
@@ -0,0 +1,12 @@
+angular.module('statusBar', [])
+
+.directive('statusBar', function() {
+  return {
+    restrict: 'E',
+    scope: {
+      msgText: '='
```

Comment:
I didn't know about `component()` yet... thank you for pointing it out! I'll have a look and see if it makes sense. If not, I agree with your idea of using `@` rather than `=`.

- [ ] look into using `component()` here